src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
```

```suggestion
.stars__star {
```
No need to increase the specificity